test/unit/helpers/xhr/labels_login_ja.js

```diff
@@ -3,6 +3,7 @@ define({
   'responseType': 'json',
   'response': {
     'enroll.call.setup': 'JA: enroll.call.setup',
-    'security.disliked_food': 'JA: What is the food you least liked as a child?'
+    'security.disliked_food': 'JA: What is the food you least liked as a child?',
+    'oform.errorbanner.title': 'Japanese error banner title'
```

Comment:
Doesn't this need the ```JA: ``` prefix?

Reply:
We don't need the JA prefix. My goal for this test is just to make sure that I am validating the text value for the key `oform.errorbanner.title` that is coming from login_ja file. Essentially mocking the behavior we get when we have bundles.login loaded with japanese translations.